### PR TITLE
Reverse translit implementation

### LINKED ISSUE #61: reverse translit
Are any ways to use reverse translit?
`pryvit_svite` => `привіт світе`

### PR DESCRIPTION
**Pull request by sasha-kir**
Hello! I found your package looking for reverse transliteration and since I didn't find anything, I decided to try to implement it myself. This should resolve #61 

I had to separate digraphs from simple letters, so I made some significant changes to the letter mappings. I also made ц > ts, х  > kh, щ > shch shared for Russian and Ukrainian to simplify a bit (there are many standards for Russian that support that, according to [wikipedia](https://ru.wikipedia.org/wiki/%D0%A2%D1%80%D0%B0%D0%BD%D1%81%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8F_%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%BE%D0%B3%D0%BE_%D0%B0%D0%BB%D1%84%D0%B0%D0%B2%D0%B8%D1%82%D0%B0_%D0%BB%D0%B0%D1%82%D0%B8%D0%BD%D0%B8%D1%86%D0%B5%D0%B9))

I also noticed issues with uppercase conversion and word boundaries (first letter associations applied only to the first letter of the whole string), so I fixed those.

I hope I'm not out of line here. This is actually my first pull request 😬

### CODE REVIEW

**Comment by greybax** [comment]
@sasha-kir thank you for your PR! Looks pretty good for the first sight. I will take a look at this PR more deeply on the weekend.

Just one thing is backward compatibility: 1 word could have lot of transliteration variations. How we can handle it?

**Comment by sasha-kir** [comment]
That is true. My code correctly handles only translit produced by the same rules as transform function (e.g. it doesn’t handle ja > я, only ya > я). Do you think reverse function should handle all possible variations?

I also made some simplifications (e.g. in Russian “i“ will always be “и” - there is also “ы” and “й”, but they are way less frequent).

I think it’s not possible to correctly reverse all translit back to cyrillic without a dictionary :( In my implementation I tried to do my best.

File: test/test.js [hunk 1]
```diff
@@ -25,17 +25,41 @@ describe('cyrillicToTranslit', () => {
...
+    });
+    it('should transform digraphs to uppercase correctly', () => {
+      assert.equal("Yaroslavl i Yekaterinburg", cyrillicToTranslit().transform("Ярославль и Екатеринбург"));
     });
     it('should return translit with replaced " " on "_"', () => {
       assert.equal("privet_mir!", cyrillicToTranslit().transform("привет мир!", "_"));
     });
     it('should return translit if translit passed', () => {
       assert.equal("privet mir!", cyrillicToTranslit().transform("privet mir!"));
     });
```
Comment (greybax) [inline test/test.js]: That’s awesome!

**Comment by greybax** [comment]
Looks good to me! Thank you again for you PR @sasha-kir! Great job especially for the first PR ;) 

PR is approved

**Comment by greybax** [comment]
Will bump package to new major version and update readme accordingly.

**Pull request merged**